packages/headless/src/controllers/core/folded-result-list/headless-core-folded-result-list.test.ts

```diff
@@ -108,14 +150,14 @@ describe('FoldedResultList', () => {
       ).toBeFalsy();
     });
 
-    it('dispatches analytics and folding/loadCollection when calling loadCollection', () => {
-      foldedResultList.loadCollection(foldedResultList.state.results[0]);
-      expect(engine.actions.pop()?.type).toEqual(
-        'analytics/folding/showMore/pending'
-      );
-      expect(engine.actions.pop()?.type).toEqual(
-        'folding/loadCollection/pending'
-      );
+    it('#loadCollection dispatches #loadCollection AND #logShowMoreFoldedResults analytics', () => {
+      const expectedLogShowMoreAction = 'analytics/folding/showMore/pending';
+      const expectedLoadCollectionAction = 'folding/loadCollection/pending';
+
+      foldedResultList.loadCollection(foldedResultList?.state?.results[0]);
+      expect(foldedResultList.state.results).not.toBe(undefined);
```

Comment:
why do we need this expect here? we are testing  the dispatch of `loadCollection` and `logShowMoreFoldedResults` no?

Reply:
oh I tried to add this before I moved the test, I can remove it as I thought it could fix the test failing originally.